Antidote/CallsManager.m

```diff
@@ -271,6 +275,43 @@ - (void)ringingCallDeclineButtonPressed:(RingingCallViewController *)controller
                             error:nil];
 }
 
+#pragma mark - ActiveCallViewControllerDataSource
+
+- (NSInteger)activeCallControllerNumberOfPausedCalls:(ActiveCallViewController *)controller
+{
+    return [[self.allPausedCallsController fetchedObjects] count];
+}
+
+- (NSString *)activeCallController:(ActiveCallViewController *)controller pausedCallerNicknameForCallAtIndex:(NSIndexPath *)indexPath
+{
+    OCTCall *call = [self.allPausedCallsController objectAtIndexPath:indexPath];
+
+    OCTFriend *friend = [call.chat.friends firstObject];
+
+    return friend.nickname;
+}
+
+- (NSTimeInterval)activeCallController:(ActiveCallViewController *)controller pauseTimeDurationForCallAtIndex:(NSIndexPath *)indexPath
+{
+    OCTCall *call = [self.allPausedCallsController objectAtIndexPath:indexPath];
+
+    return call.callDuration;
```

Comment:
I was thinking of a timer :smile: . Though `onHoldStartDate` could work, though I'm not sure what would be the best way to refresh the tableView. Also I thought saving realm objects, you lose precision on time with NSDate.